cmd/kubelet/app/server.go

```diff
@@ -785,8 +787,14 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 	}
 
 	if s.HealthzPort > 0 {
+		checks := []healthz.HealthChecker{healthz.PingHealthz}
```

Comment:
I think this should go here https://github.com/kubernetes/kubernetes/blob/3aff1f97bef13f1d40e47fae785994d57c3e5a80/pkg/kubelet/server/server.go#L363-L367 with the other handlers

Reply:
They are for different servers (port 10250 vs 10248). The port 10248 is purely for /healthz controlled by `--healthz-port` while the other I believe is the main kubelet server including most endpoints.

Although two servers both have /healthz endpoint, they have different lists of checks (10248 has only PingHealthz), so I wondered if we should add new one for both of them.

https://github.com/kubernetes/kubernetes/pull/63812 seemed to recommend using dedicated healthz port (10248)

Reply:
wow, thanks, I didn't notice that